pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -322,22 +361,84 @@ func (c *podIPMetricsCollector) setupDirectoryWatcher(dir string) error {
 	return nil
 }
 
+func (c *podIPMetricsCollector) countTotalIPs(subnet *net.IPNet) (uint64, error) {
```

Comment:
Chatted with both Jingyuan and Yining on this - let's explore a bit more on reusing the existing library from the OSS CNI repo.

Reply:
Done